src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  const activeNav
+  = ({ isActive }: { isActive: boolean }) => classNames('navbar-item', {
+    'has-background-grey-lighter': isActive,
+  });
```

Comment:
move it outside the component (you can put before `Navigation` component) because this is hardcoded code which will never change so there is no need to create it each time per each render in the component